extras/docker/qns/run_endpoint.sh

```diff
@@ -47,18 +45,15 @@ check_testcase () {
 }
 
 run_quiche_client_tests () {
-    for req in $REQUESTS
-    do
-        # get path only from the url
-        file=$(echo $req | perl -F'/' -an -e 'print $F[-1]')
-        $QUICHE_DIR/$QUICHE_CLIENT $QUICHE_CLIENT_OPT \
-            $CLIENT_PARAMS $req > $DOWNLOAD_DIR/$file 2> $LOG || exit 127
-    done
+    sleep 3
```

Comment:
Is this required? Maybe add a comment to say why?

Reply:
https://github.com/marten-seemann/quic-interop-runner/issues/61
I added a comment